src/collective/lazysizes/tests/test_upgrades.py

```diff
@@ -93,33 +93,9 @@ def setUp(self):
 
     def test_upgrade_to_2_registrations(self):
         version = self.setup.getLastVersionForProfile(self.profile_id)[0]
-        self.assertGreaterEqual(version, self.to_version)
+        self.assertGreaterEqual(int(version), int(self.to_version))
         self.assertEqual(self.total_steps, 2)
 
-    @unittest.skipIf(IS_PLONE_5, 'No easy way to test this under Plone 5')
```

Comment:
if you remove this test we can't guarantee the upgrade step work; I prefer to remove it completely and warn users on the changelog.

Reply:
This upgrade step just [import ``jsregistr`` and cook resources](https://github.com/collective/collective.lazysizes/blob/master/src/collective/lazysizes/upgrades/v4/configure.zcml).

It will not break the way it is now.